src/SWP/Bundle/ContentBundle/Resources/config/serializer/CmfRoutingBundle/Model.Route.yml

```diff
@@ -0,0 +1,21 @@
+Symfony\Cmf\Bundle\RoutingBundle\Model\Route:
```

Comment:
we have that class: `SWP\Bundle\ContentBundle\Document\Route`. You created it ;) But jms serializer needs config for every extended class.